Packs/Core/Playbooks/playbook-A_Successful_login_from_TOR.yml

```diff
@@ -0,0 +1,802 @@
+        tooltip: ""
+        readonly: false
+      title: Select containment & Eradication steps
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 9e1167c7-f6f3-4230-8db0-ef2f65c915b7
+    type: condition
+    task:
+      id: 9e1167c7-f6f3-4230-8db0-ef2f65c915b7
+      version: -1
+      name: User Exists And Disable User Account Selected?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "12"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: Select containment & Eradication steps.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Disable the user account
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: ActiveDirectory.Users.sAMAccountName
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 08e40ba9-ff3b-4bdf-809c-914a919a54fa
+    type: condition
+    task:
+      id: 08e40ba9-ff3b-4bdf-809c-914a919a54fa
+      version: -1
+      name: User Exists And Expire User Password Selected?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "15"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: Select containment & Eradication steps.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Expire the user password (Active Directory Only)
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: ActiveDirectory.Users.sAMAccountName
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -460,
+          "y": 830
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: b31d6055-ca01-444c-8587-c9b76b4fed78
+    type: condition
+    task:
+      id: b31d6055-ca01-444c-8587-c9b76b4fed78
+      version: -1
+      name: Block Traffic From TOR Exit Nodes Selected?
```

Comment:
I think a better name is, Block Traffic from TOR Exist Nodes? Selected seems redundant if the user chooses yes then we will go here, if selected no, then this flow is not executed so there is no need to indicate "Selected"